src/app/destinyTrackerApi/d2-reviewDataCache.ts

```diff
@@ -140,13 +151,17 @@ class D2ReviewDataCache {
       this._setMaximumTotalVotes(bulkRankings);
 
       bulkRankings.forEach((bulkRanking) => {
-        const matchingScore = this._getMatchingItemByReviewKey(bulkRanking);
+        const cachedItem = this._getMatchingItemByReviewKey(bulkRanking);
+
+        if (cachedItem) {
+          const updatedCachedItem = produce(cachedItem, (newCachedItem) => {
+            newCachedItem.fetchResponse = bulkRanking;
+            newCachedItem.lastUpdated = new Date();
+            newCachedItem.overallScore = this._getScore(bulkRanking);
+            newCachedItem.ratingCount = bulkRanking.votes.total;
+          });
```

Comment:
You don't need immer for this - it's equivalent to:

```js
const updatedCachedItem = {
  ...cachedItem.
  fetchResponse: bulkRanking,
  lastUpdated: new Date(),
  overallScore: this._getScore(bulkRanking),
  ratingCount = votes.total
};
```

Immer really shines when you have deeply nested objects that you want to mess with.